nodejs/db2util/callDb2util.js

```diff
@@ -0,0 +1,100 @@
+const { Client } = require('ssh2');
+
+function callDb2util(config, done) {
+    const {
+      verbose = false,
+    } = config;
+  
+    const client = new Client();
+    const command = `/QOpenSys/pkgs/bin/db2util "${config.query}"`;
```

Comment:
should/can the query be enclosed in single quotes instead of double quotes to avoid shell processing of the text?